templates/settings.html

```diff
@@ -187,7 +187,7 @@ <h1 class="page-header">
                     <div class="form-group row" id="curpassword_group">
                         <label class="col-2 col-form-label">Current Password</label>
                         <div class="col-3">
-                            <input class="form-control" name="curpassword" type="password" value="">
+                            <input class="form-control" name="current-password" type="password" value="">
```

Comment:
this should also be changed in `settings.py:settings_page()`

Reply:
fixed